docs/general/installation/truenas.md

```diff
@@ -0,0 +1,534 @@
+To see the current location of these (and other mounts to the container), click the folder icon on the bottom-right as well in the **Workloads** widget.
+
+![Volume Mount Info Opened](/images/docs/install-truenas-6.png)
+
+Once opened, you will see a list of all mounts and their paths.
+You can use this info to navigate to them on your system shell.
+Note that you will need root (sudo) access to reach Docker-related directories on SCALE.
+
+## Managing Container Images
+
+![App Settings Config Dropdown](/images/docs/install-truenas-7.png)
+
+If you want to see the exact images installed on your TrueNAS SCALE server, go to the main Apps page, then click on the `Configuration` button.
+On the dropdown menu, click on **Manage Container Images**.
+You will be brought to a new page where you will see a list of all installed Docker images.
+
+You can also pull new images to use in the future.
+
+![Pull Image GUI](/images/docs/install-truenas-19.png)
+
+You can choose to pull from Jellyfin's [Docker Hub](https://hub.docker.com/r/jellyfin/jellyfin) or [GitHub Container Registry](https://github.com/jellyfin/jellyfin/pkgs/container/jellyfin).
+
+- To pull from Docker Hub, use: `jellyfin/jellyin`
+- To pull from from GHCR, use: `ghcr.io/jellyfin/jellyfin`
+- [You can check this blog post for more info about GHCR:](https://jellyfin.org/posts/jellyfin-release-10.9.0#key-release-notesbreaking-changes)
+  - "Docker users: We now offer GitHub Container Registry (GHCR) as an alternative container registry in addition to Docker Hub. You can pull images from the new registry via URIs like `ghcr.io/jellyfin/jellyfin:latest`. Don't worry, we have no plans to drop Docker Hub as a container registry, but we feel providing both gives users more choice and flexibility."
+- Also check out [this forum post](https://forum.jellyfin.org/t-new-jellyfin-server-web-release-10-9-6?pid=25895#pid25895) about how Docker image tags can be used.
+
+If you have a container using any of your images, SCALE should notify you of an available update to the image through its GUI provided you used the **latest** tag, or one of the other tags that doesn't point to a single release.
+
+## NVIDIA GPUs on SCALE (v24.10+)
+
+- For users with an NVIDIA GPU, [read the v24.10 Electric Eel release notes regarding your GPU:](https://www.truenas.com/docs/scale/24.10/gettingstarted/scalereleasenotes/#upgrade-notes)
+  - `"Starting in 24.10, TrueNAS does not include a default NVIDIA GPU driver and instead provides a simple NVIDIA driver download option in the web interface. This allows for driver updates between TrueNAS release versions."`
+  - `"Users can enable driver installation from the Installed applications screen. Click Configure > Settings and select Install NVIDIA Drivers. This option is only available for users with a compatible NVIDIA GPU and no drivers installed or for users who have previously enabled the setting."`
```

Comment:
Please add instructions on how to setup HWA on Intel and AMD on Truenas scale

Reply:
Do you mean adding instructions for how to do so within the Jellyfin dashboard?

If not, unless it's adding drivers manually to Scale, this is not needed.

For the Scale app launcher GUI, all you need to do to pass the Intel and AMD GPUs to the container is to click on the GPU option on the bottom of the launcher as shown in the document.

I also provided commented YAML lines for the same HWA passthrough.

This specific block is added because Scale no longer packages the NVIDIA GPU drivers and users need to install them separately using some built-in options in Scale.